backend/dataall/core/permissions/services/tenant_policy_service.py

```diff
@@ -1,20 +1,13 @@
-import logging
+from dataall.core.permissions.db.tenant.tenant_policy_repositories import TenantPolicy
+from dataall.core.permissions.constants import permissions
```

Comment:
Instead of importing all permissions, we could import only the one that is used TENANT_ALL (Look at suggestion in overall comments)